src/main/MainArray.java

```diff
@@ -0,0 +1,41 @@
+package main;
+
+import model.Resume;
+import storage.ArrayStorage;
+
+import java.util.logging.Logger;
+
+public class MainArray {
+    private static final Logger log = Logger.getLogger("MainArray Logger");
+    static ArrayStorage arrayStorage = new ArrayStorage();
+
+    public static void main(String[] args) {
+        Resume r1 = new Resume("uuid1");
+        Resume r2 = new Resume("uuid2");
+        Resume r3 = new Resume("uuid3");
+
+        arrayStorage.save(r1);
+        arrayStorage.save(r2);
+        arrayStorage.save(r3);
+
+        arrayStorage.update(r1);
```

Comment:
How can you see that the update method actually works? Consider adding something to Resume, i.e. title and update it

Reply:
Done